apps/google-analytics-4/frontend/src/components/config-screen/map-account-property/MapAccountPropertySection.tsx

```diff
@@ -85,15 +79,19 @@ export default function MapAccountPropertySection(props: Props) {
             <Select.Option key="empty option" value="" isDisabled>
               Select a "[Analytics Account] {'>'} [Property]" option...
             </Select.Option>
-            {flattenedProperties.map((flattenedProperty: FlattenedPropertiesType) => {
-              return (
-                <Select.Option
-                  key={flattenedProperty.propertyId}
-                  value={flattenedProperty.propertyId}>
-                  {`${flattenedProperty.accountName} (${flattenedProperty.accountId}) > ${flattenedProperty.propertyName} (${flattenedProperty.propertyId})`}
-                </Select.Option>
-              );
-            })}
+            {sortedAccountSummaries.map((accountSummary: AccountSummariesType) => (
+              <optgroup
+                label={`${accountSummary.displayName} (${getIdOnly(accountSummary.account)})`}
+                key={getIdOnly(accountSummary.account)}>
```

Comment:
note the `<optgroup>` addition here is the biggest change to clean this UI up for our users☝️